src/utils/sources-tree/getURL.js

```diff
@@ -60,7 +64,7 @@ export function getURL(sourceUrl: string): { path: string, group: string } {
         // with a weird URL. Just group them all under an anonymous group.
         return merge(def, {
           path: url,
-          group: "(no domain)",
+          group: defaultDomain,
```

Comment:
since this was fixed by devtools-html/devtools-core#698 we might not need this change. @jasonLaster thoughts?

Reply:
@codehag i still think it is a good practice to scope URLs. For instance, an eval w/ sourceURL, which is in one of the pages.

Reply:
@jasonLaster cool, this is good to go from my side then

the `data` protocol issue can be investigated separately